csp/impl/types/typing_utils.py

```diff
@@ -77,6 +80,10 @@ class CspTypingUtils39(CspTypingUtils37):
         # To support PEP 585
         _GENERIC_ALIASES = (typing._GenericAlias, typing.GenericAlias)
 
+        @classmethod
+        def is_generic_container(cls, typ):
```

Comment:
I think it's worth keeping this here, even though the 3.8 logic will work, because when we drop 3.8 support, we can just delete the 3.8 function, and use this function instead for 3.9